### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by Thebasestar** [gekkekoe/esphome-ecodan-hp#17]
Hi ,  while waiting for the esp ordered i'm trying to translate this in Italian if ok for gekkekoe, found helpfull the ecodan sd tool that already include many language, if anyone like to translate as well this is the link to the last version with ftc 7 and 6 https://library.mitsubishielectric.co.uk/files/download/SDCardFTC7 (on same page also previous FTC)

**Comment by gekkekoe** [comment]
yes please, make a pr with Italian translation. it should be isolated (just one file needed with the labels)

grazie mille!
